### PR TITLE
update example and instructions

### PR DESCRIPTION
**Pull request by raimondi1337**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,34 @@
...
+# Uploadcare Plugin for Chat Engine
 
-Easily add image uploads to chatrooms using UploadCare online service.
+Adds the ability to upload images in ChatEngine.Chats via UploadCare service
+
+### Quick Start
+
+0. Have ChatEngine instantiated and connected, and have a channel you want to upload images on
+```javascript
+const CE = ChatEngineCore.create({
```
Comment (ianjennings) [inline README.md]: Would personally prefer the constant to be called ```ChatEngine``` for branding purposes. It seems weird that it's called ChatEngine but there are no variables called that.

Maybe we should just move from ```ChatEngineCore``` to ```ChatEngine``` and then use ```CE``` as the preferred variable in apps.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,34 @@
-# Uploadcare Binding for PubNub Chat Engine
+# Uploadcare Plugin for Chat Engine
 
-Easily add image uploads to chatrooms using UploadCare online service.
+Adds the ability to upload images in ChatEngine.Chats via UploadCare service
```
Comment (ianjennings) [inline README.md]: Add a link to Uploadcare?

File: example/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# UploadCare Example Application
```
Comment (ianjennings) [inline example/README.md:1]: This is sweet, and very helpful.

File: example/index.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+    <script src="./node_modules/chat-engine/dist/latest/chat-engine.js" type="text/javascript"></script>
+    <script src="./node_modules/chat-engine-uploadcare/dist/chat-engine-uploadcare.js" type="text/javascript"></script>
+    <script>
+      UPLOADCARE_LOCALE = "en";
+      UPLOADCARE_TABS = "file url facebook gdrive dropbox instagram evernote flickr skydrive";
+      UPLOADCARE_PUBLIC_KEY = "19126fa911640117d6d6";
+    </script>
+    <script charset="utf-8" src="//ucarecdn.com/libs/widget/2.10.3/uploadcare.full.min.js"></script>
+    <script src="script.js" type="text/javascript"></script>
```
Comment (ianjennings) [inline example/index.html]: Thank you for isolating the important stuff.

File: example/index.html [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+        </div>
+    </div>
+
+
+    <script type="text/javascript" src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
+
+    <script src="./node_modules/chat-engine/dist/latest/chat-engine.js" type="text/javascript"></script>
+    <script src="./node_modules/chat-engine-uploadcare/dist/chat-engine-uploadcare.js" type="text/javascript"></script>
+    <script>
+      UPLOADCARE_LOCALE = "en";
```
Comment (ianjennings) [inline example/index.html]: I think these variables should be in ```script.js``` with a comment explaining what they are and how to get em.

File: README.md [hunk 6]
```diff
@@ -1,3 +1,34 @@
...
+
+### Quick Start
+
```
Comment (ianjennings) [inline README.md]: I think there also needs to be a link to Uploadcare setup webpage here so people know how to add the uploadcare part of the plugin. Or we bundle their upload jazz into this plugin. WDYT?

File: example/package.json [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+    "chat-engine": "^0.4.2",
+    "chat-engine-uploadcare": "0.0.3",
+    "http-server": "^0.10.0"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "SEE LICENSE IN LICENSE"
```
Comment (ianjennings) [inline example/package.json]: lawl

File: example/script.js [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+    };
+
+    // hook up the enter key for maximum usability
+    submit = function (e) {
+
+        if (e.keyCode === 13) {
+            send();
+        }
+    };
```
Comment (ianjennings) [inline example/script.js:77]: doesn't get simpler than this

File: src/plugin.js [hunk 9]
```diff
@@ -16,7 +16,7 @@ Include the [Uploadcare](https://uploadcare.com) Javascript files and configure
 </script>
 <script charset="utf-8" src="//ucarecdn.com/libs/widget/2.10.3/uploadcare.full.min.js"></script>
 
-<script src="dist/ocf-uploadcare.js" type="text/javascript"></script>
+<script src="dist/chat-engine-uploadcare.js" type="text/javascript"></script>
```
Comment (ianjennings) [inline src/plugin.js]: thanks, can't believe we're still tracking these down.

**Review by ianjennings — APPROVED** [review]
LGTM for merge. Offered some comments but changes not mandatory. Thanks for helping out with this!

File: README.md (hunk 2 above)
Comment (ianjennings) [reply README.md]: I also think this is a chance to be casual and say "ChatEngine chats" versus the literal ```ChatEngine.Chats```.

File: README.md (hunk 1 above)
Comment (raimondi1337) [reply README.md]: I was thinking the same thing, `ChatEngineCore` looks out of place.

Comment (ianjennings) [reply README.md]: Thought about this more and still agree.

```
Library: ChatEngineCore -> ChatEngine
Examples: ChatEngine -> CE
```

**Review by ianjennings — APPROVED** [review]

**Pull request merged**